ui/src/pages/user/Register.svelte

```diff
@@ -117,6 +118,12 @@
       });
   }
 
+  onMount(() => {
+    if ($user.id) {
```

Comment:
this would prevent guest users from registering into a full account

Reply:
Ah, that make sense. I've put the user ID check back the way it was.